.gitignore

```diff
@@ -52,7 +52,7 @@ BenchmarkDotNet.Artifacts/
 project.lock.json
 project.fragment.lock.json
 artifacts/
-**/Properties/launchSettings.json
+#**/Properties/launchSettings.json
```

Comment:
why it's commented and not removed?

Reply:
Just followed what has been done on other projects but yes it could be removed.